.github/workflows/run-tests.yml

```diff
@@ -9,18 +9,29 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        php: [7.3, 7.4, 8.0, 8.1]
+        php: [7.3, 7.4, 8.0, 8.1, 8.2]
```

Comment:
Probably want to consider dropping support for 7.3 and 7.4 now as they are EOL.

Reply:
Pre-8.0 PHP versions are still wildly used within the community. Until there is a need or a desire for newer versions, I would say we should stick to supporting the older ones.

Besides, this way we can support older versions of Laravel Socialite and Laravel, too.

Reply:
As Laravel has dropped support for older versions, I’d recommend we do the same and then tag a major version. That way if a developer is using an older PHP or Laravel version they can still use the package but without the latest features.

Reply:
We should do this on a different branch and properly port the codebase to PHP 8.0.